reference/media-types.md

```diff
@@ -387,6 +387,42 @@ responds to `foobar`.
 Praxis media types can declare attributes that are ordered collections of other
 media types or other Attributor types.
 
+### Praxis::Collection
+
+Praxis includes a helper, `Praxis::Collection.of(media_type)`, for working with  collections of a given media type. (*Note*: Previously, `Praxis::Collection.of` supported types that did not derive from `MediaType`. This usage has been deprecated and will be removed in a future version. Please use `Attributor::Collection.of` directly instead.)
+
+This defines a `<media_type>::Collection` type that is an `Attributor::Collection.of(<media_type>)`, that also includes  the `MediaTypeCommon` module. By default, Praxis will take the identifier of `<media_type>` and append a "collection=true" suffix to it.
+
+For example, your blog media type might have an attribute that is a collection of posts:
+
+{% highlight ruby %}
+class Blog < Praxis::MediaType
+  attributes do
+    attribute :id, Integer
+    attribute :posts, Praxis::Collection.of(Post)
+  end
+end
+{% endhighlight %}
+
+Praxis will look for a `Post::Collection` class to use for the `posts` attribute above, or create one if it does not exist. In the common case, the Praxis-defined `<media_type>::Collection` will be sufficient for any collection of media type instances. You can, however, define your own explicitly and Praxis will use that instead.
+
+For example, if you want to use a specific identifier for collections of posts that differs from the one Praxis would assume:
```

Comment:
would change sentence to:

However, you can easily create the inner class yourself it you want to use a  a specific identifier for collections of posts that differs from the one Praxis would. Here's an example of how to do that, if you wanted to have the `Posts::Collection` to have "application/vnd.acme.posts" identifier instead.